ndb_api/db_helpers.go

```diff
@@ -305,6 +305,31 @@ func (a *PostgresRequestAppender) appendProvisioningRequest(req *DatabaseProvisi
 	return req, nil
 }
 
+func (a *PostgresHARequestAppender) appendProvisioningRequest(req *DatabaseProvisionRequest, database DatabaseInterface, reqData map[string]interface{}) (*DatabaseProvisionRequest, error) {
```

Comment:
You only moved this right? Fine with me!

Reply:
Yes just moved it to be next to all the other appendProvisioningRequest methods. I THINK that is what he was asking for in one of his comments but not sure